build/index.html

```diff
@@ -534,48 +526,67 @@ <h5 class="head-5 text--white margin-bottom-medium">Create a customer</h5>
+        <strong class="is-block text--navy">expires_in</strong>
         <span class="is-block text--transparent text-3">string</span>
-        <span class="text--transparent text-3">optional, default is &ldquo;standard&rdquo;</span>
+        <span class="text--transparent text-3">optional</span>
       </td>
       <td>
-        The customer&rsquo;s plan determines content accessibility. Values can be one of the following strings: <code>free</code>, <code>standard</code>.
+        This parameter is used for letting our system know when a user should lose access. This can not be updated so we do not recommend using this for managing renewals as it could lead to service interruptions for your customers.
         <br><br>
-        This should be set depending on your product type. Use <code>free</code> if you are using subscription that is free with registration. In any other case, you should use <code>standard</code>.
+        Valid formats are a number followed by a unit of time. For example, <code>3-months</code>, <code>1-years</code>, <code>5-weeks</code>, <code>7-days</code>.
       </td>
     </tr>
     <tr class="text-2 border-bottom border--light-gray">
       <td class="nowrap">
-        <strong class="is-block text--navy">expires_in</strong>
+        <strong class="is-block text--navy">is_rental</strong>
+        <span class="is-block text--transparent text-3">integer</span>
+        <span class="text--transparent text-3">optional</span>
+      </td>
+      <td>
+        Set this to <code>1</code> to indicate this customer only has rental access to this product. Pass in an <code>expires_in</code> param to set an expiration time as well.
+      </td>
+    </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td>
+        <strong class="is-block text--navy">name</strong>
         <span class="is-block text--transparent text-3">string</span>
         <span class="text--transparent text-3">optional</span>
       </td>
+      <td>The customer&rsquo;s full name (first and last).</td>
+    </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td class="nowrap">
+        <strong class="is-block text--navy">password</strong>
+        <span class="is-block text--transparent text-3">string</span>
+        <span class="text--transparent text-3">optional, default is &ldquo;standard&rdquo;</span>
+      </td>
       <td>
-        This parameter is used for letting our system know when a user should lose access. This can not be updated so we do not recommend using this for managing renewals as it could lead to service interruptions for your customers.
-        <br><br>
-        Valid formats are a number followed by a unit of time. For example, <code>3-months</code>, <code>1-years</code>, <code>5-weeks</code>, <code>7-days</code>.
+        This must be at least eight characters long. Omitting this field will prompt the Customer to set their password the next time they sign in.
```

Comment:
I think we typically have customer as lowercase:
```suggestion
        This must be at least eight characters long. Omitting this field will prompt the customer to set their password the next time they sign in.
```